package-lock.json

```diff
@@ -14,6 +14,7 @@
         "axios": "^1.4.0",
         "carbon-components-react": "^8.29.1",
         "d3": "^7.8.4",
+        "next": "^13.4.4",
```

Comment:
May I know why are you installing next js ? @farisyasabri

Reply:
I think that one because I was wanted to try something but forgot to delete it before commit. I think we can just omit it from the code.